tests/performance/local/htts_v2/htts2_hpx.cpp

```diff
@@ -120,13 +120,22 @@ struct hpx_driver : htts2::driver
 
             if (all_count != 1)
             {
-                register_work(
-                        hpx::util::bind(&hpx_driver::wait_for_tasks
-                                  , std::ref(*this)
-                                  , std::ref(finished)
-                                   )
-                      , nullptr, hpx::threads::pending
-                      , hpx::threads::thread_priority_low);
+                hpx::threads::thread_init_data data1(
+                        hpx::threads::make_thread_function_nullary(
+                            hpx::util::bind(&hpx_driver::wait_for_tasks
+                                      , std::ref(*this)
+                                      , std::ref(finished)
+                                       ))
+                      , nullptr, hpx::threads::thread_priority_low);
+                register_work(data1);
+                hpx::threads::thread_init_data data2(
+                        hpx::threads::make_thread_function_nullary(
+                            hpx::util::bind(&hpx_driver::wait_for_tasks
+                                      , std::ref(*this)
+                                      , std::ref(finished)
+                                       ))
+                      , nullptr, hpx::threads::thread_priority_low);
+                register_work(data2);
```

Comment:
What's the rationale of duplicating this code?

Reply:
Very good question... there's none. Good catch!